arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,916 @@
+    },
+    "Workspace": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Workspace id",
+          "type": "string"
+        },
+        "name": {
+          "description": "Workspace name",
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type",
+          "type": "string"
+        },
+        "properties": {
+          "description": "Property bag",
+          "type": "object"
+        }
+      }
+    },
+    "WorkspaceCollection": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource id",
+          "type": "string"
+        },
+        "name": {
+          "description": "Workspace collection name",
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type",
+          "type": "string"
+        },
+        "location": {
+          "description": "Azure location",
+          "type": "string"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/AzureSku"
+        },
+        "properties": {
+          "description": "Properties",
+          "type": "object"
+        }
+      }
+    },
+    "CreateWorkspaceCollectionRequest": {
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Azure location"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/AzureSku"
+        }
+      }
+    },
+    "UpdateWorkspaceCollectionRequest": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/AzureSku"
+        }
+      }
+    },
+    "CheckNameRequest": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Workspace collection name"
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type",
+          "default": "Microsoft.PowerBI/workspaceCollections"
+        }
+      }
+    },
+    "CheckNameResponse": {
+      "type": "object",
+      "properties": {
+        "nameAvailable": {
+          "type": "boolean",
+          "description": "Specifies a Boolean value that indicates whether the specified Power BI Workspace Collection name is available to use."
+        },
+        "reason": {
+          "type": "string",
+          "description": "Specifies the reason that a Power BI Workspace Collection name cannot be used. Property is not set if the specified name is available."
+        },
+        "message": {
```

Comment:
Seems like reason and message do the same thing when the name is not available.
- reason: Specifies the reason that a Power BI Workspace Collection name cannot be used
- message: Indicates why the specified name is unavailable

Please document the difference correctly in the description.